modules/admin_manual/pages/configuration/server/harden_server.adoc

```diff
@@ -330,7 +330,8 @@ protocol = tcp
 filter = owncloud
 maxretry = 3
 bantime = 10800
-logpath = /var/owncloud_data/owncloud.log
+# If running 'Owncloud Server', try instead: /var/lib/univention-appcenter/apps/owncloud/data/files/owncloud.log
```

Comment:
# If you're running 'Owncloud Appliance', try instead: /var/lib/univention-appcenter/apps/owncloud/data/files/owncloud.log

Reply:
yes. that looks appropriate.